### DISCUSSION TITLE
uninstall Fermata Control

### DISCUSSION THREAD

**Discussion by Dikinak** [AndreyPavlenko/Fermata#150] [Q&A]
After several updates I have 3 icons in Android Auto for Fermata Control but no one working. 

When I uninstall Fermata Auto icons Fermata Control stayed.

How can I uninstall Fermata Control and Fermata Auto? 
I would like to make a clean install. 

Thank you for help.

**Comment by Alejandro2454** [comment]
Para desinstalar  fermata  control ve a ajustes, aplicaciones. Buscar=
fermata control Te aparecerá fermata control, Seleccionas y desinstalas

**Reply by Dikinak** [reply]
Thank you very much :) I am stupid that is very easy solution.
